fab/const.py

```diff
@@ -18,6 +18,7 @@
     'django_celery',
     'django_pillowtop',
     'formsplayer',
+    'formplayer',
```

Comment:
`formsplayer` & `formplayer` that's just confusing, no wonder you didn't notice it before